internal/controller/controllers/bmh_agent_controller_test.go

```diff
@@ -992,6 +1097,7 @@ var _ = Describe("bmac reconcile", func() {
 			host = newBMH(bmhName, &bmh_v1alpha1.BareMetalHostSpec{Image: image, BootMACAddress: macStr, BMC: bmh_v1alpha1.BMCDetails{CredentialsName: fmt.Sprintf(adminKubeConfigStringTemplate, clusterName)}})
 			annotations := make(map[string]string)
 			annotations[BMH_AGENT_IGNITION_CONFIG_OVERRIDES] = `{"ignition":{"version":"3.1.0", "security": {"tls":{"certificateAuthorities":[{"source":"data:text/plain;charset=utf-8;base64,c29tZSBjZXJ0aWZpY2F0ZQ=="}]}}}}`
+			annotations[BMH_CLUSTER_REFERENCE] = fmt.Sprintf(`{"name":"%s","namespace":"%s"}`, clusterName, testNamespace)
```

Comment:
Why set this in these tests? Wouldn't we want this to continue testing the behavior as it was before?

Reply:
Removed